src/components/canvas.tsx

```diff
@@ -39,20 +37,14 @@ class Canvas extends React.Component<CanvasProps, CanvasState> {
   draw() {
     if( this.props.furniture) {
       const canvas = this.refs.canvas as HTMLCanvasElement;
-      this.state.blobDebounce.clear();
       this.state.card.draw(canvas, this.props.furniture)
         .then(() => {
-          this.state.blobDebounce.clear();
-          this.state.blobDebounce(canvas, this.props.update);
+          this.props.update(canvas);
         })
         .catch( error =>  console.log(error) );
     }
   }
 
-  updateBlob(canvas: HTMLCanvasElement, update: (blob? : Blob) => void){
-    canvas.toBlob(blob => update(blob || undefined))
```

Comment:
By not calling to blob after each draw, we remove unnecessary overhead and improve performance. Handling updates in this way, should also prevent outdated versions of the canvas being downloaded / uploaded as images.